django_saml2_auth/urls.py

```diff
@@ -7,4 +7,5 @@
     url(r'^acs/$', views.acs, name="acs"),
     url(r'^welcome/$', views.welcome, name="welcome"),
     url(r'^denied/$', views.denied, name="denied"),
+    url(r'^finish-logout/$', views.finish_signout, name="denied"),
```

Comment:
name shouldn't be copy-pasted like that :)

Reply:
🤦‍♂️